tests/tf/spaces/test_dict_space.py

```diff
@@ -0,0 +1,45 @@
+"""This script tests garage.tf.spaces.dict functionality."""
+import unittest
+
+from tests.envs.dummy import DummyDictEnv
+
+from garage.misc import ext
+from garage.tf.envs import TfEnv
+
+
+class TestDictSpace(unittest.TestCase):
+    def test_dict_space(self):
+        ext.set_seed(0)
+
+        # A dummy dict env
+        dummy_env = DummyDictEnv()
+        dummy_act = dummy_env.action_space
```

Comment:
Shouldn't this be dummy_env.observation_space?

Reply:
Ignore this. I found you set action_space to dict as well. Sorry.

Reply:
I was under the impression we can't sample from observation_space b/c some observation spaces will be from -inf to inf, which is not able to be sampled.